src/server.rs

```diff
@@ -980,7 +1020,7 @@ where
 
 impl<T, B: IntoBuf> Future for Handshake<T, B>
     where T: AsyncRead + AsyncWrite,
-          B: IntoBuf,
+          B: IntoBuf + 'static,
```

Comment:
What necessitated the addition of the static bound?

Reply:
Yea, I think this additional bounds would be a breaking change?

Reply:
without it:

```
:; cargo check
   Compiling h2 v0.1.2 (file:///Users/ver/b/rs/h2)
error[E0310]: the associated type `<B as bytes::IntoBuf>::Buf` may not live long enough
    --> src/server.rs:1079:19
     |
1079 |                 c.set_target_window_size(sz);
     |                   ^^^^^^^^^^^^^^^^^^^^^^
     |
     = help: consider adding an explicit lifetime bound `<B as bytes::IntoBuf>::Buf: 'static`...
note: ...so that the type `<B as bytes::IntoBuf>::Buf` will meet its required lifetime bounds
    --> src/server.rs:1079:19
     |
1079 |                 c.set_target_window_size(sz);
     |                   ^^^^^^^^^^^^^^^^^^^^^^

error: aborting due to previous error

error: Could not compile `h2`.

```

Reply:
fwiw, the clientside handshake already had `B::Buf: 'static,`